src/Components/test/testassets/BasicTestApp/BasicTestApp.csproj

```diff
@@ -16,6 +16,11 @@
     
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- Enable RespectRequiredConstructorParametersDefault to test compatibility -->
+    <RuntimeHostConfigurationOption Include="System.Text.Json.Serialization.RespectRequiredConstructorParametersDefault" Value="true" />
+  </ItemGroup>
```

Comment:
This needs to be defined in all test projects that reference the `Microsoft.NET.Sdk.Web` and the `Microsoft.NET.Sdk.WebAssembly` the test projects live under `src/Components/tests/testassets`. It's not enough for the setting to be defined on a referenced project.

Reply:
For example, is not enough that it's defined on BasicTestApp. Components.TestServer also needs to define it.